controllers/patient.controller.js

```diff
@@ -11,6 +11,16 @@ const get = async (req, res) => {
   }
 };
 
+const getPatient = async (req, res) => {
+  try {
+    const patient = await patientServices.getPatient(req.params);
```

Comment:
anything can be passed here. use destruction to extract `slug` then send it as a single param:
```
const { slug } = req.params;

... patientService.getPatientBySlug(slug)
```